### DISCUSSION TITLE
Image URL using Nautilus

### DISCUSSION THREAD

**Discussion by milesstoetzner** [outline/outline#1816] [Self Hosting]
Hello, 

I want to embed an image using an URL on PopOS (Linux distribution based on Ubuntu).
For example,  `https://www.getoutline.com/images/screenshot.png`

On windows 10 it works to use `/image` and paste in the image URL into the file explorer.
But the file explorer of PopOS refuses to open the URL.
I assume this applies not only to PopOS but to Nautilus in general.

Am I missing something or how can I achieve this?

Windows:
![Unbenannt](https://user-images.githubusercontent.com/29414738/104813105-3dd02a80-5807-11eb-94dc-136b68889b79.png)

PopOS:

![Screenshot from 2021-01-16 14-25-42](https://user-images.githubusercontent.com/29414738/104813025-bbe00180-5806-11eb-942b-c673592e598e.png)

**Comment by tommoor** [comment]
I'm afraid I don't really follow, are you trying to embed an image hosted on Outline into the OS somehow? Where do you think the bug lies?

**Reply by milesstoetzner** [reply]
I want that the following html to be rendered in a page.

```
<img src="https://www.getoutline.com/images/screenshot.png" />
```

The img src should be any arbitrary URL. 
I do not want to upload the image to outline first.
How to do that?

When using the command `/image` then you can upload an image to outline and embed it into your page.

On windows one can pass an URL into the file explorer instead of uploading an image and outline renders the image behind the URL correctly in the page.
This does not work in my linux system.

**Reply by tommoor** [reply]
Ah, I see – you can use the markdown syntax to achieve this, it would be nice to have some UI. eg.

`![image title](https://remoteurl.png)`

**Reply by milesstoetzner** [reply]
Ah okay.  Can we implement this the same way as e.g. google docs integration but with using an img instead of iframe?
